### PR TITLE
velociraptor/add NetstatEnriched

### PR DESCRIPTION
**Pull request by mgreen27**
Adding Enriched Netstat artifact with verbose query selection options.

### CODE REVIEW

File: artifacts/testdata/windows/volatile.in.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Queries:
+  - SELECT Name, Path, CommandLine, 
+      Hash, Username, Authenticode, 
+      Family, Type, Status,
+      Laddr.IP, Laddr.Port,
+      Raddr.IP, Raddr.Port
+    FROM Artifact.Windows.Network.NetstatEnriched()
+    WHERE Name = nc.exe
```
Comment (scudette) [inline artifacts/testdata/windows/volatile.in.yaml]: I think this should be Name = "nc.exe"

Comment (mgreen27) [reply artifacts/testdata/windows/volatile.in.yaml]: done.

init.cmd should of also been: start cmd /c "artifacts\testdata\files\nc.exe -L -p 3889 -s 0.0.0.0"

File: artifacts/testdata/windows/init.cmd [hunk 2]
```diff
@@ -1,2 +1,2 @@
 mklink "c:\users\link" "c:\Windows"
-start "artifacts\testdata\files\nc.exe -L -p 3889 -s 0.0.0.0"
\ No newline at end of file
+start cmd /c "artifacts\testdata\files\nc.exe -L -p 3889 -s 0.0.0.0"
```
Comment (mgreen27) [inline artifacts/testdata/windows/init.cmd]: @scudette I cant see this line eecuting in appveyor console, should I add this somewhere else?

It looks like this mlink command is acutaly being executed from here https://github.com/Velocidex/velociraptor/blob/80cde4b12b840189e7eb512ca8e160a3489fee94/appveyor.yml
is init.cmd redundant?

**Comment by scudette** [comment]
Ah maybe you we right. I wanted to get rid of the link test anyway as I
took link support out of the main accessor.

Maybe add your command to the appveyor config?

File: artifacts/definitions/Windows/Network/NetstatEnriched.yaml [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            { SELECT Name FROM process WHERE PsId = Pid } as Name,
+            { SELECT Exe FROM process WHERE PsId = Pid } as Path,
+            { SELECT CommandLine FROM process WHERE PsId = Pid } as CommandLine,
+            { SELECT Hash FROM process WHERE PsId = Pid } as Hash,
+            { SELECT Username FROM process WHERE PsId = Pid } as Username,
+            { SELECT Authenticode FROM process WHERE PsId = Pid } as Authenticode,
+            FamilyString as Family,
+            TypeString as Type,
+            Status,
+            LaddrIP, LaddrPort,
```
Comment (scudette) [inline artifacts/definitions/Windows/Network/NetstatEnriched.yaml]: Change here the same `Laddr.IP AS LaddrIP`

File: artifacts/testdata/windows/volatile.in.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Queries:
+  - SELECT Name, Path, CommandLine, 
+      Hash, Username, Authenticode, 
+      Family, Type, Status,
+      Laddr.IP, Laddr.Port,
+      Raddr.IP, Raddr.Port
```
Comment (scudette) [inline artifacts/testdata/windows/volatile.in.yaml]: And here `LaddrIP`

File: artifacts/definitions/Windows/Network/NetstatEnriched.yaml (hunk 3 above)
Comment (mgreen27) [reply artifacts/definitions/Windows/Network/NetstatEnriched.yaml]: Modded accross all relevant files:
Laddr.IP AS LaddrIP, Laddr.Port AS LaddrPort,
Raddr.IP AS RaddrIP, Raddr.Port AS RaddrPort,

**Comment by scudette** [comment]
Thanks!

**Pull request merged**